client/service/src/task_manager/mod.rs

```diff
@@ -66,38 +66,46 @@ impl SpawnTaskHandle {
 		name: &'static str,
 		task: impl Future<Output = ()> + Send + 'static,
 	) {
-		self.spawn_inner(name, task, TaskType::Blocking)
+		self.spawn_inner(name, None, task,  TaskType::Blocking)
 	}
 
 	/// Helper function that implements the spawning logic. See `spawn` and `spawn_blocking`.
 	fn spawn_inner(
 		&self,
 		name: &'static str,
+		subsystem: Option<&'static str>,
 		task: impl Future<Output = ()> + Send + 'static,
 		task_type: TaskType,
 	) {
 		if self.task_notifier.is_closed() {
 			debug!("Attempt to spawn a new task has been prevented: {}", name);
-			return
+			return;
 		}
 
 		let on_exit = self.on_exit.clone();
 		let metrics = self.metrics.clone();
+		// Provide a default subsystem name.
+		let subsystem_name = subsystem.unwrap_or("substrate-unspecified");
```

Comment:
```suggestion
		let subsystem_name = subsystem.unwrap_or("");
```

Isn't empty working as well?

Reply:
> What do you think of naming it `group` instead of subsystem? Because in the end it is something to group different tasks?

Sounds better, indeed we just want to group by.

Reply:
Yes, it works.